ui/client/components/graph/node-modal/editors/expression/StringEditor.tsx

```diff
@@ -1,15 +1,15 @@
 import React from "react"
 import Input from "../field/Input"
 import _ from "lodash"
```

Comment:
Do you really use lodash? Maybe you can use some ts syntax? If we really need lodash please then import only these components which are used.

Reply:
agree, but this change has nothing to logic in this file, it's only effect of rename of another file. removal of Lodash can cause some problems not connected to scope of this pr.

Reply:
I've not dived into this code, the only thing which changed is th the import here